README.md

```diff
@@ -69,12 +70,34 @@ RIBASIM_DLL_REGRESSION='${IMOD_COLLECTOR_REGRESSION}/ribasim/bin/libribasim.dll'
 
 - The tests can then be run with:
 
-```bash
+```sh
 pixi run tests
 ```
 
 - Lint the codebase with:
 
-```bash
+```sh
 pixi run lint
 ```
+
+- When developing with visual studio code, it is recommended to open the application via `open-vscode.bat`.
+  This will open the application in a new vscode window with the correct environment variables set.
+
+### Debugging
+
+When debugging the unit tests in visual studio code with the test explorer, you can encounter some problems.
+MODFLOW is not so keen on multi-threading, and when debugging a test, make sure that you only have one test running at a time.
+
+When running the unit tests, this is not the case, since there is a switch statement that determines if we should call subprocess.run(), or stay within the main thread.
```

Comment:
```suggestion
When you only run, not debug, the tests, this is not the case, since there is a switch statement that determines if we should call `subprocess.run()`, or stay within the main thread.
```